sherpa/astro/xspec/tests/test_xspec.py

```diff
@@ -539,17 +538,86 @@ def test_xsxset_get(self):
         self.assertEqual('somevalue', xs.get_xsxset('Foobar'))
 
 
-if __name__ == '__main__':
-    import os
-    import sys
-    import sherpa.astro.xspec as xs
-    from sherpa.utils import SherpaTest
+@requires_xspec
```

Comment:
Because I used it in the early stages of development to test the initial implementation, but then I added an actual model in `xspec/__init__.py` so I didn't think this test served any purpose. However, commenting out the test reduces test coverage by one line which was covered when the test is run with xspec 12.9.0, the only case when this test is not skipped.

When we'll drop support for 12.9.0 the test will not be run anymore.

I now think the best option is to change the test in such a way it always runs, so I'm going to change the version constraint to 99.9.9.